src/language-graphql/printer-graphql.js

```diff
@@ -275,10 +275,6 @@ function genericPrint(path, options, print) {
       ]);
     }
 
-    case "TypeExtensionDefinition": {
```

Comment:
This AST is no longer used.

https://github.com/graphql/graphql-js/blob/58aaa1df30b501b67d6986ea2fc1a8b95ac9fbca/src/language/ast.js#L144